<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XHCMonitorDialog</class>
 <widget class="QDialog" name="XHCMonitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XHC-HB04 Monitor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="resultsTable">
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextEdit" name="textEdit"/>
      <widget class="QWidget" name="">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pb_Hide">
          <property name="text">
           <string>Hide</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Monitor">
          <property name="text">
           <string>Monitor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_ShowHaluiPins">
          <property name="text">
           <string>halui Pins</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_ShowUtilPins">
          <property name="text">
           <string>Util Pins</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_ShowXHCPins">
          <property name="text">
           <string>XHC Pins</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
